### PR TITLE
fix: adjust build to stencil default

### LINKED ISSUE #1190: check why new versions 12.7.1 does not run on stackblitz angular
- [ ] change the folder components back to dist/components

### PR DESCRIPTION
**Pull request by hirsch88**
Closes #1190

## Acceptance Criteria

- Design Review by @Gagne87, @clastzoo
- Technical Review by @hirsch88, @mladenplaninicic
- [Style Guide](https://stenciljs.com/docs/style-guide) Review @hirsch88, @mladenplaninicic
- Visual test done by @hirsch88, @mladenplaninicic, @Gagne87, @clastzoo
- Functional test done by @hirsch88, @mladenplaninicic
- Browser Testing by @hirsch88, @mladenplaninicic, @Gagne87, @clastzoo
    - Desktop
        - Chrome
        - Edge
        - Safari
        - Firefox
    - Tablet
        - iPad (Landscape / Portrait)
    - Mobile
        - Safari iOS
        - Chrome Preview

### CODE REVIEW

File: packages/components/stencil.config.ts [hunk 1]
```diff
@@ -10,10 +10,11 @@ export const config: Config = {
   ...StencilBaseConfig,
   buildEs5: 'prod',
   extras: {
+    dynamicImportShim: true,
     initializeNextTick: true,
     scriptDataOpts: true,
-    appendChildSlotFix: true,
-    cloneNodeFix: true,
+    // appendChildSlotFix: true,
```
Comment (hirsch88) [inline packages/components/stencil.config.ts]: - [x] add them again

File: packages/components/config/stencil.bindings.vue.ts [hunk 2]
```diff
@@ -41,5 +41,5 @@ export const VueGenerator = (
     componentCorePackage,
     componentModels: vueComponentModels,
     excludeComponents,
-    customElementsDir: 'dist/components',
+    customElementsDir: 'components',
```
Comment (hirsch88) [inline packages/components/config/stencil.bindings.vue.ts]: - [x] rename back to dist

File: packages/components/config/stencil.basic.config.ts [hunk 3]
```diff
@@ -32,6 +32,15 @@ export const StencilBaseConfig: Config = {
   outputTargets: [
     {
       type: 'dist-custom-elements',
+      dir: 'components',
```
Comment (hirsch88) [inline packages/components/config/stencil.basic.config.ts]: - [x] remove dir and use default

File: packages/components/config/stencil.basic.config.ts [hunk 4]
```diff
@@ -13,16 +13,16 @@ if (IS_BAL_DS_RELEASE) {
...
+  autoprefixCss: true,
   namespace: 'design-system-components',
   hashedFileNameLength: 10,
-  sourceMap: false,
+  // sourceMap: false,
   globalStyle: 'src/styles/global.sass',
   globalScript: 'src/global.ts',
   watchIgnoredRegex: [/\.stories\.(js|jsx|ts|tsx|mdx)$/, /\/stories\//], // ignore storybook files in --watch mode
-  enableCache: true,
+  // enableCache: true,
```
Comment (hirsch88) [inline packages/components/config/stencil.basic.config.ts]: - [x] remove cache

File: packages/components/config/stencil.basic.config.ts [hunk 5]
```diff
@@ -13,16 +13,16 @@ if (IS_BAL_DS_RELEASE) {
 }
 
 export const StencilBaseConfig: Config = {
+  autoprefixCss: true,
   namespace: 'design-system-components',
   hashedFileNameLength: 10,
-  sourceMap: false,
+  // sourceMap: false,
```
Comment (hirsch88) [inline packages/components/config/stencil.basic.config.ts]: - [x] remove source maps

File: tsconfig.base.json [hunk 6]
```diff
@@ -8,23 +8,23 @@
...
     "moduleResolution": "node",
+    "target": "es2017",
     "noImplicitAny": true,
     "noImplicitReturns": true,
     "noUnusedLocals": false,
     "noUnusedParameters": true,
     "pretty": true,
     "removeComments": true,
-    "inlineSources": true,
+    // "inlineSources": true,
```
Comment (hirsch88) [inline tsconfig.base.json]: - [x] remove them

File: .gitignore [hunk 7]
```diff
@@ -98,6 +98,7 @@ cypress/videos
 .vercel/
 
 # Design System
+packages/components/components
```
Comment (hirsch88) [inline .gitignore]: - [x] remove this path

File: packages/components/config/stencil.basic.config.ts [hunk 8]
```diff
@@ -32,6 +30,14 @@ export const StencilBaseConfig: Config = {
   outputTargets: [
     {
       type: 'dist-custom-elements',
+      copy: [
+        {
+          src: '../config/custom-elements',
+          dest: 'components',
```
Comment (hirsch88) [inline packages/components/config/stencil.basic.config.ts]: - [x] change to dist/components

**Review by mladenplaninicic — APPROVED** [review]

**Pull request merged**